src/devTools.js

```diff
@@ -68,7 +80,7 @@ class DevToolsEnhancer {
       type,
       id: this.socket.id,
       name: this.instanceName,
-      instanceId: this.appInstanceId,
+      instanceId: this.instance.id,
```

Comment:
I appreciate the critical thinking here. That said, I'd love if we could make this patch smaller so it will be easier to maintain. Would it have any negative effect to undo the following customizations you made:

1. Undoing the customization to use an object for `this.instance` instead of tracking things separately in `this.instanceName`, `this.instanceId`, etc.
2. Undoing the merging of `this.instanceId` and `this.appInstanceId`

Reply:
applied according changes